tensorflow_recommenders_addons/dynamic_embedding/core/kernels/lookup_impl/lookup_table_op_gpu.h

```diff
@@ -60,6 +64,16 @@ class TableWrapperBase {
   virtual void dump(K* d_key, ValueType<V>* d_val, const size_t offset,
                     const size_t search_length, size_t* d_dump_counter,
                     cudaStream_t stream) const {}
+  virtual Status save_to_file_system(OpKernelContext* ctx,
```

Comment:
I recommend you put the logic of Status and OpKernelContext to this API's caller( upper level ) to do it. Just an advice.

Reply:
Fix. Put ctx outside but status need to be retained, cause filesystem api will return status.